core/src/dird/ua_select.cc

```diff
@@ -866,12 +866,11 @@ bool SelectMediaDbrByName(UaContext* ua,
                           const char* volumename)
 {
   new (mr) MediaDbRecord();
-  POOLMEM* err = GetPoolMemory(PM_FNAME);
+  std::string err{};
```

Comment:
If err string should be used like a buffer, err needs to be resized to the required size. Otherwise it stays zero.

Reply:
same as above, no need for resizing.